cmd/util.go

```diff
@@ -133,13 +133,21 @@ func getOwnerCapture(ctx context.Context) (*capture, error) {
 	return nil, errors.NotFoundf("owner")
 }
 
-func applyAdminChangefeed(ctx context.Context, job model.AdminJob) error {
+func applyAdminChangefeed(ctx context.Context, job model.AdminJob, credential *security.Credential) error {
 	owner, err := getOwnerCapture(ctx)
 	if err != nil {
 		return err
 	}
-	addr := fmt.Sprintf("http://%s/capture/owner/admin", owner.AdvertiseAddr)
-	resp, err := http.PostForm(addr, url.Values(map[string][]string{
+	scheme := "http"
```

Comment:
Need a new helper function for creating the Client?

Reply:
We have a helper function already, `httputil.NewClient`.